src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksVendorToKfsVendorDetailConversionServiceImpl.java

```diff
@@ -83,9 +83,17 @@ protected KfsVendorDataWrapper createVendorDetail(PaymentWorksVendor pmwVendor,
             kfsVendorDataWrapper = buildRemainingVendorNotes(pmwVendor, kfsVendorDataWrapper);
             kfsVendorDataWrapper.getVendorDetail().setBoNotes(kfsVendorDataWrapper.getVendorNotes());
         }
+        validatePaymentWorksVendorCustomFields(pmwVendor, kfsVendorDataWrapper);
```

Comment:
The PMW vendor cannot really be processed at all if the PMW vendor type is null. 
Therefore; for consistency, the validation check for PMW vendor type being null should probably be
in class : edu.cornell.kfs.pmw.batch.service.impl.PaymentWorksNewVendorRequestsServiceImpl
in method: canPaymentWorksNewVendorRequestProcessingContinueForVendor

Reply:
OK, I shall change it